modules/platform/nuxeo-platform-web-common/src/main/java/org/nuxeo/ecm/platform/web/common/requestcontroller/filter/NuxeoRequestControllerFilter.java

```diff
@@ -70,13 +72,36 @@
 
     protected static final int LOCK_TIMEOUT_S = 120;
 
+    protected static final String IDEMPOTENCY_KEY = "NuxeoIdempotencyKey";
```

Comment:
Could we use something more generic, like `Idempotency-Key` or `X-Idempotency-Key` maybe?

Reply:
Of course, I was not sure if we wanted this header to be Nuxeo-specific so i just went with this value for now

Reply:
I moved the key in the code but we should still discuss naming, so keeping this conversation open

Reply:
`Idempotency-Key` is good it's what's in the draft RFC and used by lots of organizations already.